CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+# dbt_iterable_source v0.8.1
+[PR #30](https://github.com/fivetran/dbt_iterable_source/pull/30) introduces the following updates: 
+
+## Bug Fixes
+- Updated `stg_iterable__user_unsub_message_type` to partition on `message_type_id` along with `email`. This brings in all unsubscriptions for the `unique_user_key` across all message types for an email when the `user_unsubscribed_message_type_history` source table is brought into the staging model.
+- Updated `stg_iterable__user_unsubscribed_channel` and `stg_iterable__user_unsubscribed_channel` to partition on `channel_id` along with `email`. This brings in all unsubscription for the `unique_user_key` across all channels for an email  when the `user_unsubscribed_channel_history` source table is brought into the staging model.
```

Comment:
It is reading a bit confusing when you mention that these changes are for when the `user_unsubscribed_message_type_history` or `user_unsubscribed_channel_history` source tables are brought into the staging model. Could we instead mention that these changes are only impacting connectors setup before August 2023?